src/main/shell-session/shell-session.ts

```diff
@@ -40,41 +41,140 @@ export class ShellOpenError extends Error {
   }
 }
 
+export enum WebSocketCloseEvent {
```

Comment:
Not sure if we need all events defined (documented) here because we are just using few of them?

Reply:
I decided to include all of them so that were could use the name and have some docs for them in our code.

I don't see the point of only including a few of them in an enumeration but if you think it would be better to just document the uses I can revert.

I prefer enums over commented (and especially uncommented) special values.